examples/basic/variables.tf

```diff
@@ -0,0 +1,8 @@
+variable "project_id" {
+  type = string
+  description = "The Project ID where to create the Folder"
```

Comment:
Folders are not created inside projects.

Reply:
removed